src/module/migration/migrations/825-khakkhara.ts

```diff
@@ -0,0 +1,41 @@
+import { ItemSourcePF2e } from "@item/data";
+import { MigrationBase } from "../base";
+
+/** Update from khakkara to khakkhara */
+export class Migration825Khakkhara extends MigrationBase {
+    static override version = 0.825;
+
+    override async updateItem(source: ItemSourcePF2e): Promise<void> {
+        if (source.img === "systems/pf2e/icons/equipment/weapons/khakkara.webp") {
+            source.img = "systems/pf2e/icons/equipment/weapons/khakkhara.webp";
+        }
```

Comment:
```suggestion
        if (source.img.endsWith("icons/equipment/weapons/khakkara.webp")) {
            source.img = source.img.replace("khakkara.webp", "khakkhara.webp");
        }
```
In case of Forge rearranging system file paths